README.md

```diff
@@ -8,13 +8,13 @@ For full details on ThreatExchange and best practices are available in the [Thre
 
 The `api-reference-examples` folder contains example implementations in various languages for using the API. These implementations are at various stages of completeness and may not all implement every endpoint availible on the ThreatExchange API. For full details on the ThreatExchange API and UI, data formats, and best practices are available in the [ThreatExchange docs](https://developers.facebook.com/docs/threat-exchange/).
 
-## pytx3
+## python-threatexchange
 
-`pytx3` is an all-in-one python package CLI which allows basic integration with ThreatExchange including matching on publically availible data.
+An all-in-one python Library/CLI tool available on pypi under `threatexchange` which allows basic integration with ThreatExchange including matching on publicly available data.
```

Comment:
Clarification worth making? 
'publicly available to apps with ThreatExchange access'